### PR TITLE
check connection to all databases

### PR DESCRIPTION
**Pull request by javierwilson**

### CODE REVIEW

File: corehq/apps/hqadmin/service_checks.py [hunk 1]
```diff
@@ -157,10 +159,27 @@ def check_heartbeat():
 
 
 def check_postgres():
+    connected = True
+    status_str = ""
+    for db in settings.DATABASES:
+        db_conn = connections[db]
+        try:
+            c = db_conn.cursor()
```
Comment (dimagimon) [inline corehq/apps/hqadmin/service_checks.py]: F841 local variable 'c' is assigned to but never used

Comment (javierwilson) [inline corehq/apps/hqadmin/service_checks.py]: cheeky!

**Review by emord — APPROVED** [review]
Looks good. You may want to add some newlines though. Also is there a way we could use this for the standbys as well? Are they in settings.DATABASES?

**Review by esoergel — APPROVED** [review]
Looks good.  I agree with Emord that you'll probably want some newlines in there, though.  You could also make these two separate checks - one to test the connections, and one to perform a dummy query.

**Comment by javierwilson** [comment]
I am going to merge this now, pending improvements.  Standbys are not listed anywhere in the Django environment :( But it is kind of urgent to test standys and backups. I will come up with a solution for this.

**Pull request merged**